scanpipe/pipes/purldb.py

```diff
@@ -313,15 +314,37 @@ def get_unique_unresolved_purls(project):
     for item in distinct_unresolved:
         pkg_type, namespace, name, extracted_requirement = item
         if range_class := RANGE_CLASS_BY_SCHEMES.get(pkg_type):
+            purl = PackageURL(type=pkg_type, namespace=namespace, name=name)
+
             try:
                 vers = range_class.from_native(extracted_requirement)
-            except InvalidVersionRange:
-                continue
+            except InvalidVersionRange as exception:
+                details = {
+                    "purl": purl,
+                    "extracted_requirement": extracted_requirement,
+                }
+                project.add_error(
+                    description="Invalid VersionRange",
+                    model="get_unique_unresolved_purls",
+                    details=details,
+                    exception=exception,
+                )
+
+            except InvalidVersion as exception:
+                details = {
+                    "purl": purl,
+                    "extracted_requirement": extracted_requirement,
+                }
+                project.add_error(
+                    description="Invalid exctracted requirement",
+                    model="get_unique_unresolved_purls",
+                    details=details,
+                    exception=exception,
+                )
```

Comment:
Could we remove duplication?

Since on the description changes, what about using:
```
except (InvalidVersionRange, InvalidVersion) as exception:
```